tutorials-v5/visualization/animation-demo.md

```diff
@@ -0,0 +1,214 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Animation demos
+
+For more information about QuTiP see [http://qutip.org](http://qutip.org)
+
+```python
+from qutip import (ket, basis, sigmaz, tensor, qeye, mesolve,
+                   complex_array_to_rgb, about)
+import qutip as qt
+import numpy as np
+import matplotlib.pyplot as plt
+```
+
+
+```python
+# if this does not work on your environment, use qt.make_html_video
+# to see animations.
+%matplotlib notebook
+```
+
+## Time evolution of an operator
+
+
+```python
+# Hamiltonian
+H = qt.sigmay().unit()
+
+# initial state
+psi0 = qt.sigmax().unit()
+
+# list of times for which the solver should store the state vector
+tlist = np.linspace(0, 2*np.pi, 50)
+
+results = mesolve(H, psi0, tlist, [], [])
+
+fig, ani = qt.hinton(results.states)
+
+```
+
+
+## Time evolution of a ket
+
+
+```python
+# Hamiltonian
+H = qt.rand_dm(5)
```

Comment:
Try for this example something like a displaced cavity?

```
#cavity annihilation operator
N_cut = 5
a = destroy(N_cut)

#cavity Hamiltonian with constant displacement
H = a.dag() * a + 0.5 * (a+a.dag())

# initial state
psi0 = basis(N_cut, 0)

# list of times for which the solver should store the state vector
tlist = np.linspace(0, 10, 100)

c_ops = [np.sqrt(0.1)  * a]

results = mesolve(H, psi0, tlist, c_ops, [])

```

You might need to play with the parameters a little bit, to get something reasonable.  Also you could consider making the displacement term time-dependent (just a cosine pre-factor on the a+a.dag() term)